pootle/core/search/backends/elasticsearch.py

```diff
@@ -75,6 +92,10 @@ def search(self, unit):
             }
         )
 
+        # Sometimes urllib seems to return empty string so handle this here
```

Comment:
Can you share a sample log in the issue, please? I might be wrong, but isn't this a potential bug in urllib3? That's what I was referring to with masking an error, and the handling being a workaround. Keeping that in mind, I don't believe adding extra logging here solves the root cause.